manifests/params.pp

```diff
@@ -31,6 +31,12 @@
     $service_enable = true
   }
 
+  if ($::operatingsystem == 'Ubuntu' and $::operatingsystemrelease in ['15.04','15.10','16.04','16.10']) or ($::operatingsystem == 'Debian' and $::operatingsystemrelease == '8') {
```

Comment:
Heh, I think it depends if you're using Ubuntu upstreams or Puppet's ones 😄 

Either way, Util resolution is the right way 👍